DigitalLearningSolutions.Web/Views/FindYourCentre/Index.cshtml

```diff
@@ -1,10 +1,15 @@
-﻿@using DigitalLearningSolutions.Web.Helpers
-@using DigitalLearningSolutions.Web.ViewModels.FindYourCentre
+﻿@using DigitalLearningSolutions.Web.ViewModels.FindYourCentre
 @model FindYourCentreViewModel
 @{
   ViewData["Title"] = "Find Your Centre";

```

Comment:
Should this be capitalised like this? Doesn't match the heading capitalisation

Reply:
According to @stellake , the capitalisation here is correct, but the consistency isn't too important. 

"So H1 should be only the first letter capitalised
But titles are opposite and usually have every first letter capitalised
But I wouldn't worry too much about it, the titles are only in the tab and mainly for screenreader users"

Reply:
OK so we're just inconsistent with uncapitalised titles on half the pages then, but this one is correct.